modules/rtmessageadd.php

```diff
@@ -42,6 +42,14 @@
 	if($message['destination']!='' && $message['sender']=='customer')
 		$error['destination'] = trans('Customer cannot send message!');
 
+    $ticketcontent = $LMS->GetTicketContents($message['ticketid']);
```

Comment:
A może to w całości wstawić do bloku if {} ? Po co to ma się wykonywać jak opcja jest nieaktywna?

Reply:
Super pomysł.